runtime/lua/man.lua

```diff
@@ -729,6 +733,8 @@ function M.open_page(count, smods, args)
 
   if not ok then
     error(ret)
+  elseif vim.fn.exists('g:pager') == 1 then
+    set_options(vim.g.pager)
```

Comment:
If `g:pager` is a global it doesn't need to be set in `set_options`, so you can remove the argument from `set_options` and just change this to `set_options()`.

Reply:
GitHub does not let me comment on unchanged lines, but also change `set_options(true)` on line 669 to

```lua
vim.g.pager = true
set_options()
```

Reply:
That does seem like a more clean solution, but `g:pager` now does not exist in normal Man buffers. I don't know whether this is an ok behavior. Previously it is set to false.

Reply:
It's fine if `g:pager` doesn't exist in normal Man buffers.